src/v2/BaseLoginRouter.js

```diff
@@ -129,26 +81,18 @@ export default Router.extend({
   },
 
   handleIdxResponseSuccess (idxResponse) {
-    const { interactionCode } = idxResponse;
-    if (interactionCode) {
-      const authClient = this.settings.getAuthClient();
-      const codeVerifier = this.settings.get('codeVerifier');
-      this.settings.unset('codeVerifier');
-      return authClient.token.exchangeCodeForTokens({ codeVerifier, interactionCode })
-        .then(({ tokens }) => {
-          this.settings.callGlobalSuccess(Enums.SUCCESS, { tokens });
-        })
-        .catch(err => {
-          this.settings.callGlobalError(err);
-        });
+    if (idxResponse.interactionCode) {
+      return completeLoginFlow(this.settings, idxResponse);
```

Comment:
nit: Since this requires an `interactionCode` to work, we should rename this method to `completeInteration` (or something similar) avoid developers thinking this is a global complete/login handler.